ksql-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializer.java

```diff
@@ -82,6 +80,9 @@ public GenericRow deserialize(final String topic, final byte[] bytes) {
 
   private Object enforceFieldType(Schema fieldSchema, String delimitedField) {
 
+    if (delimitedField.length() == 0) {
```

Comment:
one minor suggestion string has `isEmpty` method, that has the same logic: `value.length == 0`, which personally I think makes the code more expressive here.